src/components/Header.res

```diff
@@ -1,21 +1,19 @@
 @react.component
 let make = () => {
-  <div>
-    <Dsfr.Header
-      brandTop={<>
-        {`République`->React.string}
-        <br />
-        {`Française`->React.string}
-      </>}
-      homeLinkProps={{"href": "/", "title": "Catala Explorer"}}
-      /* operatorLogo={{"alt": "Catala logo", "imgUrl": "../assets/logo.png", "orientation": "vertical"}} */
-      serviceTitle={<>
-        {"Explicabilité des algorithmes publics"->React.string}
-        <Dsfr.Badge noIcon=true severity={#success} as_=#span small=false>
-          {"beta"->React.string}
-        </Dsfr.Badge>
-      </>}
-      serviceTagline="Prototypes de simulateurs de prestations sociales avec explications invidualisées, détaillées et intelligibles"
-    />
-  </div>
+  <DSFR.Header
+    brandTop={<>
+      {React.string("République")}
+      <br />
+      {React.string("Française")}
+    </>}
+    homeLinkProps={{"href": "/", "title": "Catala Explorer"}}
+    /* operatorLogo={{"alt": "Catala logo", "imgUrl": "../assets/logo.png", "orientation": "vertical"}} */
```

Comment:
Dead code? Or should we put the logo back?

Reply:
Oh yeah, I don't know. Do we need to add the Catala logo in the header?